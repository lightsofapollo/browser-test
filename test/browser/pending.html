<!DOCTYPE HTML>
<html>
<head>
  <meta http-equiv="content-type" content="text/html; charset=utf-8" />
  <title>body</title>
</head>
<body>
<script src="/browser_test/browser_test.js" type="text/javascript" charset="utf-8"></script>
<script type="text/javascript" charset="utf-8">
browserTest.createSuite(1, 'pending.html');

browserTest.createTest('pending');
browserTest.pendingTest();

browserTest.finishSuite(new Date());
</script>
</body>
</html>


