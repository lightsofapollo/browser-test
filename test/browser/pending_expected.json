{
  "ok": true,
  "asserts": [
    {
      "ok": true,
      "number": 1,
      "name": "pending # SKIP"
    }
  ],
  "pass": [
    {
      "ok": true,
      "number": 1,
      "name": "pending # SKIP"
    }
  ],
  "fail": [],
  "todo": [],
  "errors": [],
  "plan": {
    "start": 1,
    "end": 1
  }
}